sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -235,31 +224,19 @@ def get_media_settings_value(physical_port, key):
                 helper_logger.log_error("Error: No values for physical port '{}'".format(physical_port))
             return {}
 
-        if key[VENDOR_KEY] in media_dict:
-            if is_si_per_speed_supported(media_dict[key[VENDOR_KEY]]):
-                if key[LANE_SPEED_KEY] is not None and key[LANE_SPEED_KEY] in media_dict[key[VENDOR_KEY]]:
-                    return media_dict[key[VENDOR_KEY]][key[LANE_SPEED_KEY]]
-                else:
-                    return {}
-            else:
-                return media_dict[key[VENDOR_KEY]]
-        if key[VENDOR_KEY].split('-')[0] in media_dict:
-            if is_si_per_speed_supported(media_dict[key[VENDOR_KEY].split('-')[0]]):
-                if key[LANE_SPEED_KEY] is not None and key[LANE_SPEED_KEY] in media_dict[key[VENDOR_KEY].split('-')[0]]:
-                    return media_dict[key[VENDOR_KEY].split('-')[0]][key[LANE_SPEED_KEY]]
-                else:
-                    return {}
-            else:
-                return media_dict[key[VENDOR_KEY].split('-')[0]]
-        elif key[MEDIA_KEY] in media_dict:
-            if is_si_per_speed_supported(media_dict[key[MEDIA_KEY]]):
-                if key[LANE_SPEED_KEY] is not None and key[LANE_SPEED_KEY] in media_dict[key[MEDIA_KEY]]:
-                    return media_dict[key[MEDIA_KEY]][key[LANE_SPEED_KEY]]
+        for dict_key in media_dict.keys():
```

Comment:
@DennisChiuEC can you write a function `get_media_settings`

Reply:
Done